tests/verifyPMMInventory_test.js

```diff
@@ -175,3 +175,41 @@ Scenario(
     }
   },
 );
+
+Scenario(
+  'PMM-T1226 - Verify Agents has process_exec_path option on Inventory page @inventory @exporters @pgsm-pmm-integration',
```

Comment:
@puneet0191 I suggest to put this test to nightly. This is pure UI. And we will test all exporters we have there

Reply:
if this is just UI, I think it is fine, just move it to Nightly